ow_lander/scripts/camera_capture_action_client.py

```diff
@@ -11,11 +11,8 @@
 
 parser = argparse.ArgumentParser(
   formatter_class=argparse.ArgumentDefaultsHelpFormatter,
-  description="Trigger both mast camera to capture a photograph."
+  description="Trigger both mast cameras to capture photographs."
 )
-parser.add_argument('exposure', type=float, nargs='?', default=-1,
-  help="Camera exposure in seconds. If <= 0 the previous exposure setting " \
-       "will be used.")
 args = parser.parse_args()
```

Comment:
I was mimicking other action clients that have that line. I guess I should remove it from all of them for consistency? While we're at it, I think the clients that don't take arguments don't need any argparse code at all. Is that reasonable?

Reply:
The argparse code is still what generates the help message and therefore a description of what the script does, so I think our script interface is more consistent if they all have it, even those without arguments. 

Please do not remove `**var(args)` from the other client scripts, only the ones that do not have arguments. It does not really make a difference here because its unpacking an empty namespace, and therefore passing no keyword variables to the function, but I think the intent of _passing no arguments_ is easier to read when its not there.

Reply:
Sorry, I was unclear. What I meant is that I should remove all argparse lines from scripts that do not take arguments.

Reply:
Where do you see the help message printed? I was unable to find it.

Reply:
> Sorry, I was unclear. What I meant is that I should remove all argparse lines from scripts that do not take arguments. Scripts that take arguments would remain unchanged.

Oh, no worries. Yes that I think would make sense.

> Where do you see the help message printed? I was unable to find it.

If you do `rosrun ow_lander camera_capture_action_client.py --help` you'll see 
```
usage: camera_capture_action_client.py [-h]

Trigger both mast cameras to capture photographs.

optional arguments:
  -h, --help  show this help message and exit
```

Reply:
Removed `**vars(args)` from client scripts that take no arguments. Kept `args = parser.parse_args()` in all client scripts because it is necessary to print help messages.